src/main/java/eu/mihosoft/vrl/v3d/TextExtrude.java

```diff
@@ -0,0 +1,165 @@
+package eu.mihosoft.vrl.v3d;
```

Comment:
Again, I don't like the AWT dependency. Please consider the JavaFX API. I cannot merge this PR if we don't support Android.